src/components/SimpleFooter/SimpleFooter.tsx

```diff
@@ -64,7 +67,7 @@ const SimpleFooter: FunctionComponent<SimpleFooterProps> = ({
               if (
                 utag &&
                 utag.gdpr &&
-                utag.gdpr.showConsentPreferences &&
+                utag.gdpr.showConsentPreferences("'" + language + "'") &&
```

Comment:
@babsmiles As `language` is already coming in as a string, you can just do `utag.gdpr.showConsentPreferences(language)` here.